### ISSUE TITLE
Different behaviour of extension manager and devel:badextensions script

### ISSUE THREAD

**Issue by LarsGit223** [splitbrain/dokuwiki#1236]
The extension manager checks the download URL of a plugin for updates or faults/problems.

The script behind the page devel:badextensions only checks against the master branch which is not always representing the latest release.

So it comes to wrong warnings about versions not matching which makes the page a bit useless (in my opinion).

Is there a chance to change the behaviour of the script? Is the source code accessible somewhere/is there a GitHub repository for it?

If someone gives me the code, I would like to make the adjustments.

**Comment by Klap-in** [comment]
The check is performed by this script: 
https://github.com/splitbrain/dokuwiki-plugin-pluginrepo/blob/master/helper/version.php
and https://github.com/splitbrain/dokuwiki-plugin-pluginrepo/blob/master/cron.php which calls version.php at line 49

I guess that it doesn't download the archives and checks its content for the correct dates is that just an check of a file at github is simpler and due to (lack of) trust in the real content of a downloaded archive.

But for the real reasoning it is better that @splitbrain, who implements this script, reacts on your question.

**Comment by splitbrain** [comment]
Correct. the script only accesses the plugin.info.txt file in the github
master branch. I don't want it to download archives. However it could be
adjusted to figure out the correct branch from the given github repo (eg.
when there's a stable branch or something). In the end this is a script to
give developers a hint, if it's invalid they are welcome to ignore it.

**Comment by selfthinker** [comment]
Can you open the script and put it somewhere? Then others can improve it.

**Comment by splitbrain** [comment]
@selfthinker gerrit linked the source above

**Comment by selfthinker** [comment]
Ah, sorry, I saw that but assumed it would be the check in the extension manager. I didn't think the check for devel:badextensions would happen in a plugin as well.
Reading clearly helps. ;-)
